DIPs/DIP1000.md

```diff
@@ -87,29 +88,72 @@ the function.
 
 #### Reachability vs. lifetime
 
-For each value `v` within a program we define the notion of <i>reachability</i>
+##### Reachability
+
+For each value `v` within a program we define the notion of _reachability_
 denoted as `reachability(v)`, akin to the lexical extent through which the value
 can be accessed.
 
-- For an rvalue, the reachability is the expression within which it is used.
+- For an unnamed temporary (either lvalue such as `array[f()]` or rvalue such as
+`42 + f()`), the reachability is the largest grammatical expression within which
+it is used.
 - For a named variable in a scope, the reachability is the lexical scope of the variable per the language rules.
-- For a module-level variable, reachability is considered infinite. Notation: `reachability(v)` `=` `∞`.
+- For a module-level variable, reachability is considered infinite. Notation:
+`reachability(v)` `=` `∞`.
+
+For values with finite reachability, we denote their reachability as a tuple
+`M:B:E`, where:
```

Comment:
As the parenthesized below mentions, the tuple definition clarifies when reachability starts and ends. "Lexical scope" is vague and does not apply to unnamed temporaries.